src/Android/Xamarin.Android/Samples/GeometryEngine/ClipGeometry/ClipGeometry.cs

```diff
@@ -0,0 +1,258 @@
+        }
+
+        private void ClipButton_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                // Remove the Colorado graphic from the input geometries graphics overlay collection. That way it will be easier 
+                // to see the clip versions of the of the GeometryEngine.Clip operation. 
+                _inputGeometriesGraphicsOverlay.Graphics.Remove(_coloradoGraphic);
+
+                // Create a simple line symbol for the resulting GeometryEngine.Clip operation geometries - it will be the same 
+                // as the one used for the boundary of Colorado.
+                SimpleLineSymbol clipGeometriesSimpleLineSymbol = new SimpleLineSymbol(
+                    SimpleLineSymbolStyle.Solid, System.Drawing.Color.Blue, 4);
+
+                // Create the color that will be used as the fill for the resulting GeometryEngine.Clip operation geometries. It 
+                // will be a semi-transparent, blue color and it will be the same as the one used for the boundary of Colorado.
+                System.Drawing.Color clipGeometriesFillColor = System.Drawing.Color.FromArgb(34, 0, 0, 255);
+
+                // Create the simple fill symbol for the resulting GeometryEngine.Clip operation geometries - comprised of a 
+                // fill style, fill color and outline. It will be the same as the one used for the boundary of Colorado.
+                SimpleFillSymbol clipGeometriesSimpleFillSymbol = new SimpleFillSymbol(
+                    SimpleFillSymbolStyle.Solid, clipGeometriesFillColor, clipGeometriesSimpleLineSymbol);
+
+                // Loop through each graphic in the input geometries for the clip operation.
+                foreach (Graphic oneGraphic in _inputGeometriesGraphicsOverlay.Graphics)
+                {
+                    // Perform the clip operation. The first parameter of the clip operation will always be the Colorado graphic. 
+                    // The second parameter of the clip operation will be one of the 3 different clip geometries (_outsideGraphic, 
+                    // _containedGraphic, or _intersectingGraphic).
+                    Geometry myGeometry = GeometryEngine.Clip(_coloradoGraphic.Geometry, (Envelope)oneGraphic.Geometry);
+
+                    // Only work on returned geometries that are not null.
+                    if (myGeometry != null)
+                    {
+                        // Create the graphic as a result of the clip operation using the defined symbology. 
+                        Graphic clippedGraphic = new Graphic(myGeometry, clipGeometriesSimpleFillSymbol);
+
+                        // Add the clipped graphic to the clip areas graphics overlay collection.
+                        _clipAreasGraphicsOverlay.Graphics.Add(clippedGraphic);
+                    }
+                }
+
+                // Disable the button after has been used.
+                _clipButton.Enabled = false;
```

Comment:
I'd like to see another button that would "reset" the sample so it could be run again (without having to exit and return to the sample). Maybe the "clip" button could re-enable after the user resets? Just an idea.